CSS-Networking: What is AGIC (spelling&grammar) (#127821)
Azure/azure-content: articles/application-gateway/ingress-controller-overview.md

> [!NOTE]
> Customers can only deploy one AGIC add-on per AKS cluster, and each AGIC add-on currently can only target one Application Gateway. For deployments that require more than one AGIC per cluster or multiple AGICs targeting one Application Gateway, please continue to use AGIC deployed through Helm.
>

> > Both Helm and AGIC add-on do not support ExternalName service.

Reviewer: ```suggestion
> Both Helm and AGIC add-on don't support ExternalName service.
```
Thanks for the update.
The proper fix is **don't**. Use of contractions is the Microsoft style. Please apply update before this can be merged.

Reviewer 2: @awestrem - Can you respond to the requested changes or let us know if you would like us to commit them?

@mbender-ms  - When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

Author: The requested changes are good.  Please commit.  Thank you!
